### PR TITLE
Set the default browser size explicitly before tests.

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
Set the default browser size explicitly before tests.

@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by mjhenkes** [comment]
I also removed some screenshots that got added previously at the wrong screen size.

File: config/wdio/wdio.conf.js [hunk 1]
```diff
@@ -48,4 +48,9 @@ exports.config = {
     ui: 'bdd',
     timeout: 20000,
   },
+
+  before: () => {
+    /* global browser */
+    browser.setViewportSize({ width: 1050, height: 895, name: 'default' });
```
Comment (emilyrohrbough) [inline config/wdio/wdio.conf.js]: Would we want to define this as a terra-size ? Or leave for now till MVB?

**Review by JakeLaCombe — APPROVED** [review]

File: config/wdio/wdio.conf.js (hunk 1 above)
Comment (mjhenkes) [reply config/wdio/wdio.conf.js]: the Terra variable isn't available here for some reason.

**Comment by mjhenkes** [comment]
3 min 37 sec for before hook vs 3 min 3 sec with docker-compose.yml
But thats just one run of each, hard to say if it is statistically significant.

**Comment by emilyrohrbough** [comment]
I imagine it won't have a huge impact given our current setup coninuously adject the viewport size when multiple viewports are supplied to `terra.should.beAccessible` or `terra.should.matchScreenshot`I would be curious to verify this against the core repo for tests not defining a viewport explicitly

**Comment by bjankord** [comment]
Let's move forward with the docker-compose approach.

**Review by bjankord — APPROVED** [review]

**Pull request merged**